plugins/adminhelp.sma

```diff
@@ -13,133 +13,181 @@
 
 #include <amxmodx>
 
-new g_cvarDisplayClientMessage;
-new g_cvarHelpAmount;
+const MaxMapLength         = 32;
```

Comment:
But if you search through ReHLDS for mapname, in some places it's defined with 32 (a null-terminated string). I'm confused.